File: python_programming/Notes/old_enough.py
#GA 7th Old Enough

age = int(input("How old are you: "))


if age >= 18:
    print("you are old enough to vote!")
elif age >= 16:
    print("you are old enough to drive!")
elif age >= 15:
    print("you are old enough to get a learners permit!")
elif age >= 5:
    print("you are old enough to go to school")
else:
    print("you are the youngest person ever!")

File: C_Programming/Practices/old_enough.c
//GA 7th Old Enough
#include <stdio.h>
int num;

int main(void){
    printf("how old are you?\n");
    scanf("%d", &num);
if(num == 18){
    printf("you can vote\n");
}else if (num ==16){
    printf("you can drive!\n");
}else if (num == 15){
    printf("You can get a learners permit!!!\n");
}else if (num ==4)
    printf("you can go to school!!!\n");
else{
    printf("are you the youngest person EVER!\n");
}
return 0;
}